docs/api/spec.md

```diff
@@ -181,10 +184,10 @@ Since version 1.2 of the FDC3 Standard it may do so via the [`fdc3.getInfo()`](r
 ```ts
 import {compareVersionNumbers, versionIsAtLeast} from '@finos/fdc3';
 
-if (fdc3.getInfo && versionIsAtLeast(await fdc3.getInfo(), '1.2')) {
+if (fdc3.getInfo && versionIsAtLeast(await fdc3.getInfo(), "1.2")) {
   await fdc3.raiseIntentForContext(context);
 } else {
-  await fdc3.raiseIntent('ViewChart', context);
+  await fdc3.raiseIntent("ViewChart", context);
 }
 ```
 
```

Comment:
This is not something you changed here I know, but on line 211:
  let {appId, instanceId}
is missing spaces in the braces.

Also, I've no idea why someone would do that as a let rather than const actually - it's not like these are values you'd expect to manipulate!

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85